huey/storage.py

```diff
@@ -122,6 +122,10 @@ def flush_all(self):
 class RedisStorage(BaseStorage):
     def __init__(self, name='huey', blocking=False, read_timeout=1,
                  max_errors=1000, connection_pool=None, **connection_params):
+        redis_url = connection_params.pop('url', None)
+        if redis_url:
+            connection_pool = redis.ConnectionPool.from_url(redis_url, decode_components=True)
```

Comment:
> Is the redis_url mutually exclusive with other parameters here? What is the behavior if a user erroneously supplies arguments for both url and connection_pool?

`redis_url` will take precedence even if a `connection_pool` or other `connection_params` have been specified. But such a conflict already existed with `connection_pool` and `connection_params`, where `connection_pool` wins if specified.

To fix this: Would you like a `ValueError` being raised if more than one of these parameters are specified at the same time? I'm happy to supplement this PR :)

> Also, why is url not just a regular keyword argument?

I just thought it was a good fit for `connection_params`.